src/main/java/frc/robot/Robot.java

```diff
@@ -291,37 +293,43 @@ private void spinControlPanel() {
         }
       }
 
-      String colorString;
+      String detectedColorString;
       Color detectedColor = colorSensor.getColor();
       ColorMatchResult match = colorMatcher.matchClosestColor(detectedColor);
       if (match.color == kBlueTarget) {
-        colorString = "Blue";
+        detectedColorString = "Blue";
       } else if (match.color == kRedTarget) {
-        colorString = "Red";
+        detectedColorString = "Red";
       } else if (match.color == kGreenTarget) {
-        colorString = "Green";
+        detectedColorString = "Green";
       } else if (match.color == kYellowTarget) {
-        colorString = "Yellow";
+        detectedColorString = "Yellow";
       } else {
-        colorString = "Unknown";
+        detectedColorString = "Unknown";
       }
-      detectedColorEntry.setString(colorString);
+      String lastDetectedColorString = detectedColorEntry.toString(); 
+      detectedColorEntry.setString(detectedColorString);
       confidenceEntry.setDouble(match.confidence);
-
+      
       int doubleContolPanelSpinAmount = controlPanelSpinAmount * 2;
-      if (targetControlPanelColor == "Green" && (colorString != "Green" || doubleContolPanelSpinAmount > 0)) {
-        // @TODO: Add wheel spinner code (currently waiting for more details on this).
-      } else if (targetControlPanelColor == "Red" && (colorString != "Red" || doubleContolPanelSpinAmount > 0)) {
-        // @TODO: Add wheel spinner code (currently waiting for more details on this).
-      } else if (targetControlPanelColor == "Blue" && (colorString != "Blue" || doubleContolPanelSpinAmount > 0)) {
-        // @TODO: Add wheel spinner code (currently waiting for more details on this).
-      } else if (targetControlPanelColor == "Yellow" && (colorString != "Yellow" || doubleContolPanelSpinAmount > 0)) {
-        // @TODO: Add wheel spinner code (currently waiting for more details on this).
-      }
+      turnControlPanel(detectedColorString, targetControlPanelColor, lastDetectedColorString,
+          doubleContolPanelSpinAmount);
       targetSpinEntry.setDouble(controlPanelSpinAmount);
     } else {
       detectedColorEntry.setString("N/A");
       confidenceEntry.setDouble(0);
     }
   }
+
+  /*
+   * This function turns the control panel when called upon in spinControlPanel().
+   */
```

Comment:
The parameters of this method should be documented via Javadoc notation, e.g.
```java
  /**
   * This function turns the control panel when called upon in spinControlPanel().
   * @param detectedColor The current detected color of the control panel.
   */
```
Do take note of the two asterisks on the first line, another standard of [Javadoc](https://docs.oracle.com/javase/7/docs/technotes/tools/solaris/javadoc.html#leadingasterisks):
![image](https://user-images.githubusercontent.com/13039555/74678786-08036b80-518a-11ea-9f03-6b9670c72212.png)